src/AppBundle/Form/User/UserDetailsFullType.php

```diff
@@ -31,7 +17,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         ->add('addressPostcode', 'text')
         ->add('addressCountry', 'country', [
             'preferred_choices' => ['', 'GB'],
-            'empty_value' => $this->addressCountryEmptyValue,
+            'empty_value' => 'Please select ...',
```

Comment:
it was originally injected in the constructor, but it was never edited since the start, so I reduced the complexity and left it in English inside